apps/dashboard/src/main.tsx

```diff
@@ -171,6 +172,10 @@ const router = createBrowserRouter([
                 path: ROUTES.ENVIRONMENTS,
                 element: <EnvironmentsPage />,
               },
+              {
+                path: ROUTES.WEBHOOKS,
+                element: <WebhooksPage />,
+              },
```

Comment:
do we want to hide this route by IS_SELF_HOSTED and FF?

Reply:
I want to keep it actually, since I have an Upgrade to cloud CTA there